src/main/java/uk/gov/hmcts/reform/idam/health/tokenstore/TokenStoreHealthProbeProperties.java

```diff
@@ -22,5 +22,7 @@ static class Probe {
     private TokenStoreHealthProbeProperties.Probe replication;
     private TokenStoreHealthProbeProperties.Probe workQueue;
     private TokenStoreHealthProbeProperties.Probe connections;
+    private TokenStoreHealthProbeProperties.Probe alive;
```

Comment:
[minor] Maybe we should keep the names consistent with AM - "live" (vs "alive"). Applies to the actual probe classes too.

Reply:
Yeah, FR is weirdly inconsistent between the health probes, which is why I did it like this. For DS I wanted to avoid naming the probes after the "/healthy" endpoint, because "HealthyHealthProbe" is annoying, and also in DS they use "alive" instead of "live" so I thought I'd use "alive" and "ready" to represent the two things, and do the same for AM.

Don't mind changing it, but I thought this at least made the logs/monitoring make more sense

Reply:
I don't really have a strong preference - just thought it would make the code even nicer :) 
The feign provider has the correct path anyway so not sure if naming the probes in a speicific way gives us a significant benefit to logs/monitoring.